spec/features/merchants/dashboard_spec.rb

```diff
@@ -167,19 +167,19 @@
     end
 
     scenario 'visitor sees date of invoice creation next to each item' do
-      expect(page).to have_content(@invoice_item_1.invoice_creation_date)
-      expect(page).to have_content(@invoice_item_2.invoice_creation_date)
-      expect(page).to have_content(@invoice_item_3.invoice_creation_date)
-      expect(page).to have_content(@invoice_item_4.invoice_creation_date)
-      expect(page).to have_content(@invoice_item_5.invoice_creation_date)
+      expect(page).to have_content(@invoice_1.created_at.strftime("%A, %B %d,%Y"))
+      expect(page).to have_content(@invoice_2.created_at.strftime("%A, %B %d,%Y"))
```

Comment:
If we abstract the logic to the controller these tests do not need to change. This is good.